src/components/WorkOrder/Photos/hooks/uploadFiles/compressFile.ts

```diff
@@ -12,48 +12,34 @@ function fileDetails(file: File) {
   }
 }
 
-export interface CompressResult {
-  success: boolean
-  file: File
-}
-
```

Comment:
It's easier to just have them return the File and deal with the fail handling immediately so the user doesn't waste time waiting for a process that will fail